### PR TITLE
feat: add visitor id to the flags page

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

Need an easy way for people to grab their visitor ID that are helping us test.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk

This a hidden page so not real risk.

## Testing

Going to the flags page you will see your visitor ID and be able to copy it.

## Screenshots (if applicable)
Screen Shot 2022-08-25 at 4 12 52 PM

### CODE REVIEW

File: src/pages/Flags/Debugging.tsx [hunk 1]
```diff
@@ -50,14 +70,25 @@ export const Debugging = () => {
...
+                aria-label='Copy'
+                size='sm'
+                icon={<CopyIcon />}
+                onClick={handleCopyClick}
+              />
+            </Row.Value>
+          </Row>
         </Card.Body>
+        <Card.Footer>
+          <Button onClick={() => window.location.reload()} colorScheme='blue'>
```
Comment (gomesalexandre) [inline src/pages/Flags/Debugging.tsx:87]: One less function in the stack trace = happier debugging
```suggestion
          <Button onClick={window.location.reload} colorScheme='blue'>
```

File: src/pages/Flags/Debugging.tsx [hunk 2]
```diff
@@ -1,28 +1,48 @@
...
+import { RawText } from 'components/Text'
 import { getLogLevel, saveLogLevel } from 'lib/logger'
 
 export const Debugging = () => {
   const [logLevel, setLogLevel] = useState(getLogLevel())
+  const [visitorId, setVisitorId] = useState<string | null>(null)
+
+  const handleCopyClick = async () => {
+    try {
+      await navigator.clipboard.writeText(visitorId ?? '')
```
Comment (gomesalexandre) [inline src/pages/Flags/Debugging.tsx:24]: Do we want to throw `if (!visitorId)`?

**Review by gomesalexandre — DISMISSED** [review]
Does what it says on the box, added smallish comments to be tackled at your discretion

File: src/pages/Flags/Debugging.tsx (hunk 2 above)
Comment (reallybeard) [reply src/pages/Flags/Debugging.tsx:24]: Yah prob just in case

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**